controllers/openshift_controller.go

```diff
@@ -963,7 +965,7 @@ func (r *KataConfigOpenShiftReconciler) processKataConfigInstallRequest() (ctrl.
 	}
 
 	// Create kata-oc MCP only if it's not a converged cluster
-	if machinePool == "kata-oc" {
+	if !isConvergedCluster {
```

Comment:
Is this the bug fix ? Can you elaborate ?

Reply:
Bug was maybe a bit excessive then :wink: but the change is good.

If you happen to re-push, you could maybe drop the comment on the way.

Reply:
The bug fix is the final chunk in this commit, not this one - sorry, apparently it's not as clear as I thought. ;-)

Reply:
Oh my... I really misread the changelog ! :astonished: